### ISSUE TITLE
Calendar

### ISSUE THREAD

**Issue by heroic** [SupportBee/hackerstreet.in#11]
Since we have a calendar, we can use it to allow user's to link it to the events they might be organizing. Adding a search to search by city, would also help in becoming HSI a better place for hackers to get together on.

There isn't a single resource out there that allows hackers to find events in their city. Meetup, etc don't have all hackers. HSI is where most hackers are already present, they will be able to submit events they find in their own locality, and the whole community can benefit.

**Comment by unamashana** [comment]
Yes. Absolutely. This is something I have really been wanting to add to HSI. I am going to create a new balsamiq project so we can flesh these things out. I will add you as a collaborator.
